src/platform/plugins/shared/navigation/public/solution_tour/solution_tour.ts

```diff
@@ -53,7 +53,7 @@ const SOLUTION_NAVIGATION_TOUR_KEY = 'solutionNavigationTour:completed';
 async function preserveTourCompletion(userProfile: UserProfileServiceStart): Promise<void> {
   try {
     localStorage.setItem(SOLUTION_NAVIGATION_TOUR_KEY, 'true');
-    return await userProfile.update({ [SOLUTION_NAVIGATION_TOUR_KEY]: true });
+    return await userProfile.partialUpdate({ [SOLUTION_NAVIGATION_TOUR_KEY]: true });
```

Comment:
This is confusing. I'd like "update" to be removed and "partialUpdate" to be renamed to "update."
The server side logic anyway is a partial update from what I can see. the difference is just in internal client side service state